crossdock/client/tchclient/json.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package tchclient
+
+import (
+	"time"
+
+	"github.com/uber/tchannel-go/json"
+
+	"github.com/yarpc/yarpc-go/crossdock/client/behavior"
+	"github.com/yarpc/yarpc-go/crossdock/client/random"
+)
+
+func runJSON(s behavior.Sink, call call) {
+	assert := behavior.Assert(s)
+	checks := behavior.Checks(s)
+
+	headers := map[string]string{
+		"hello": "json",
+	}
+	token := random.String(5)
+
+	resp, respHeaders, err := jsonCall(call, headers, token)
+
+	if checks.NoError(err, "json: call failed") {
+		assert.Equal(token, resp.Token, "body echoed")
+		assert.Equal(headers, respHeaders, "headers echoed")
```

Comment:
Previously we were including what was echoed. Do we not want to do that anymore?